underscore.js

```diff
@@ -1446,6 +1453,13 @@
     return _.isFunction(value) ? value.call(object) : value;
   };
 
+  // Returns the, possibly deeply nested, value at `path` if defined.
+  // Otherwise, returns `fallback`.
+  _.get = function(object, path, fallback) {
+    var value = property(path)(object);
+    return value === void 0 ? fallback : value;
+  };
```

Comment:
I mentioned `_.set`, `_.has` and `_.result` in the PR message as "Further Work". I wasn't sure if we wanted to tackle them all at once, or one at a time. (We could also pull `_.get` out of this PR if that would simplify things)

`_.unSet` was not on my radar.

To answer your question directly, I'm guessing we would be obliged, but I figured this PR would be a good forum to make that decision as a group.